### PR TITLE
feat(galoy): add notifications pod

### PR DESCRIPTION
**Pull request by bodymindarts**

### CODE REVIEW

File: charts/galoy/templates/notifications-cm.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "galoy.notifications.fullname" . }}
+data:
+  api-keys.yml: |-
```
Comment (krtk6160) [inline charts/galoy/templates/notifications-cm.yaml:6]: Should be notifications.yml or something

File: charts/galoy/templates/notifications-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  template:
+    metadata:
+      name: {{ template "galoy.notifications.fullname" . }}
+      labels:
+        app: {{ template "galoy.notifications.fullname" . }}
+        kube-monkey/enabled: enabled
+        kube-monkey/identifier: {{ template "galoy.notifications.fullname" . }}
+    spec:
+      containers:
+      - name: api-keys
```
Comment (krtk6160) [inline charts/galoy/templates/notifications-deployment.yaml:29]:
```suggestion
      - name: notifications
```

File: charts/galoy/templates/_helpers.tpl [hunk 3]
```diff
@@ -62,6 +62,14 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- default "api-keys" .Values.galoy.apiKeys.nameOverride | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{/*
+Create a default fully qualified consent name.
```
Comment (thevaibhav-dixit) [inline charts/galoy/templates/_helpers.tpl:66]: ```notifications``` name ?

File: charts/galoy/values.yaml [hunk 4]
```diff
@@ -71,6 +71,11 @@ galoy:
       ## Digest of the image
       ##
       digest: "sha256:2c5540b276d8d16c06277901a587b60344509af663b5180c7fce54d7175e4e50" # METADATA:: repository=https://github.com/GaloyMoney/galoy;commit_ref=22adf9c;app=api-keys;monorepo_subdir=core/api-keys;
+    apiKeys:
```
Comment (krtk6160) [inline charts/galoy/values.yaml:74]:
```suggestion
    notifications:
```

**Review by krtk6160 — COMMENTED** [review]
Just need to rename apiKeys to notifications in the values.yml image block, otherwise LGTM

**Pull request merged**